app/views/layouts/_adv_search_body.html.haml

```diff
@@ -73,7 +73,7 @@
                                :class             => "form-control",
                                "data-miq_focus"   => true,
                                "data-miq_observe" => {:interval => ".5", :url => url2}.to_json)
-        - if admin_user?
+        - if report_admin_user?
```

Comment:
@kbrock the checkbox here should not be limited to a report_admin_user, this checkbox should have been available to super admin or an admin user

Reply:
@kbrock i noticed this while reviewing https://github.com/ManageIQ/manageiq-ui-classic/pull/6253